<ui version="4.0" >
 <class>BasketDialog</class>
 <widget class="QDialog" name="BasketDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Basket</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="4" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Selected package(s) for install:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4" >
    <widget class="PackageView" name="packageList" >
     <property name="minimumSize" >
      <size>
       <width>450</width>
       <height>0</height>
      </size>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
     <property name="headerHidden" >
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Extra dependencies of the selected package(s) that are also going to be installed:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4" >
    <widget class="PackageView" name="extraList" >
     <property name="minimumSize" >
      <size>
       <width>450</width>
       <height>0</height>
      </size>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
     <property name="headerHidden" >
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Total Size:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QLabel" name="totalSize" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&lt;b>2.2 MB&lt;/b></string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>306</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="3" >
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>404</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3" >
    <widget class="QPushButton" name="actionButton" >
     <property name="text" >
      <string>Install Package(s)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PackageView</class>
   <extends>QTreeView</extends>
   <header location="global" >packageview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
